### ISSUE TITLE
Have comments in the changed files instead of the PR-comments

### ISSUE THREAD

**Issue by messers2** [JacobDomagala/CompileResult#25]
I've investigated the possibility to have the warnings inlined into the "changed files" as an automatic review comment instead of using the comment in the PR itself. It seems the oktokit supports something like [this](https://docs.github.com/de/github-ae@latest/rest/guides/working-with-comments).
From checking the current implementation this seems a bit more work, as these comments obviously need the line/position to be passed to the API.

Thoughts?

**Comment by JacobDomagala** [comment]
Yes this is something that could be useful. I'll try to look into it when I'll have more time. 
Thanks!
